src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,175 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
+gradlogpdf(d::Normal, x::Real) = (d.μ - x) / d.σ^2
+function logpdf(d::Normal, x::Real)
+    if iszero(d.σ)
+        d.μ == x ? Inf : -Inf
+    else
+        -(((x - d.μ) / d.σ)^2 + log2π) / 2 - log(d.σ)
+    end
+end
+function pdf(d::Normal, x::Real)
+    if iszero(d.σ)
+        d.μ == x ? Inf : 0.0
+    else
+        exp(-((x - d.μ) / d.σ)^2 / 2) * invsqrt2π / d.σ
+    end
+end
+function cdf(d::Normal, x::Real)
```

Comment:
still needs a `ccdf` implementation to deal with tails (e.g. `ccdf(Normal(),10.0)`)